internal/pkg/auth/user_token_flow.go

```diff
@@ -59,6 +60,7 @@ func (utf *userTokenFlow) RoundTrip(req *http.Request) (*http.Response, error) {
 	}
 
 	if !accessTokenValid {
+		utf.printer.Debug(print.DebugLevel, "user access token expired, reauthenticating...")
```

Comment:
```suggestion
		utf.printer.Debug(print.DebugLevel, "user access token is not valid, reauthenticating...")
```
might be something else other than expiration